scripts/cargo-update-all.sh

```diff
@@ -0,0 +1,3 @@
+cargo update --manifest-path Cargo.toml $@
+cargo update --manifest-path crates/release-automation/Cargo.toml $@
+cargo update --manifest-path crates/test_utils/wasm/wasm_workspace/Cargo.toml $@
```

Comment:
I would prefer these don't come in on code PRs. There is an automated workflow that updates the root lock file and I'd be happy if we extended that to cover the others too. But that gets tested separately from code changes is it's easier to trace where changes happened if we leave it to that job